apps/CEventService.API/DAO/EventRepository.cs

```diff
@@ -0,0 +1,35 @@
+using CEventService.API.Data;
+using CEventService.API.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace CEventService.API.DAO
+{
+    public class EventRepository : IRepository<Event>
```

Comment:
Maybe you could specify the type of repository, i.e. if it is Mongo, sqlserver, etc, in case of instantiating another event repository it would be difficult to differentiate it from this one.

Reply:
No, its not really nessesary. As in our techstack, we have decided to just one DB and only one for each service. Also remember that a microservice should have a propper database as disccussed in class. And the scope of the Event microservice does not goes beyond one database. So only one is needed. @CatCoder36

Reply:
Ok